<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>851</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="Create">
          <property name="text">
           <string>Создать</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="edit">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Редактировать</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Delete">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Удалить</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="1">
         <widget class="QLineEdit" name="name"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Название модели</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="type">
        <property name="maximumSize">
         <size>
          <width>752</width>
          <height>60</height>
         </size>
        </property>
        <property name="title">
         <string>Тип</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QRadioButton" name="cargo">
             <property name="text">
              <string>грузовой</string>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">tGroup</string>
             </attribute>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="passenger">
             <property name="text">
              <string>легковой</string>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">tGroup</string>
             </attribute>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="bus">
             <property name="text">
              <string>автобус</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">tGroup</string>
             </attribute>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="trailer">
             <property name="text">
              <string>прицеп</string>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">tGroup</string>
             </attribute>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Объём двигателя</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="engineCapacity"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Вместимость</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="capacity"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Грузоподъёмность</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="loadCapacity"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Максимальная скорость</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="maxSpeed"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Разгон до 100 км/ч</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="accelerationTo100"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QCheckBox" name="climateControl">
          <property name="text">
           <string> Наличие системы климат-контроля</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Количество</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="count"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Цена</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="price"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_12">
        <item>
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Дата последнего завоза</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="delivery">
          <property name="date">
           <date>
            <year>2022</year>
            <month>10</month>
            <day>29</day>
           </date>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="Save">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Сохранить</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <widget class="QTableWidget" name="table">
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <column>
       <property name="text">
        <string>Название модели</string>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Количество</string>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Цена (тыс. руб)</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>851</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="tGroup">
   <property name="exclusive">
    <bool>true</bool>
   </property>
  </buttongroup>
 </buttongroups>
</ui>
